test/format/format.sh

```diff
@@ -467,10 +467,13 @@ resolve()
 		# give the parent recording binary a chance to complete if we are using it
 		[[ ! -z $live_record_binary ]] && sleep 2
 
-		# Check for Sanitizer failures, have to do this prior to success because both can be reported.
-		grep -E -i 'Sanitizer' $log > /dev/null && {
-			report_failure $dir
-			continue
+		# Process group leader dumped core: so it is a bug, in contrast to spurious cores
+		# from killing zombifieed child processes. This is to guard against spuriously
+		# missing memory sanitizer errors, which has occured historically even when
+		# abort_on_error=1 was passed to MSan.
+		[[ -f "dump_t.${pid}.core" ]] {
```

Comment:
@marcbutler, I have the following error when I try to execute this code:
```
./format.sh: line 474: syntax error near unexpected token `{'
./format.sh: line 474: `                [[ -f "dump_t.${pid}.core" ]] {'
```

```suggestion
		[[ -f "dump_t.${pid}.core" ]] && {
```

Reply:
My fault ... I need to tie string around my finger and not futz with commits after testing. :)